src/fiatExchanges/FiatExchangeCurrency.tsx

```diff
@@ -142,15 +144,27 @@ function FiatExchangeCurrency({ route, navigation }: Props) {
           />
           <CurrencyRadioItem
             title="CELO"
-            selected={selectedCurrency === Currency.Celo}
-            onSelect={() => setSelectedCurrency(Currency.Celo)}
+            selected={selectedCurrency === CiCoCurrency.CELO}
+            onSelect={() => setSelectedCurrency(CiCoCurrency.CELO)}
             enabled={flow !== FiatExchangeFlow.Spend}
+            containerStyle={{
+              borderTopWidth: 0.5,
+              borderBottomWidth: 0.5,
+            }}
+            testID="radio/CELO"
+          />
+          <CurrencyRadioItem
+            title={t('celoReal')}
+            body="(cREAL)"
+            selected={selectedCurrency === CiCoCurrency.cREAL}
+            onSelect={() => setSelectedCurrency(CiCoCurrency.cREAL)}
+            enabled={flow === FiatExchangeFlow.CashIn}
             containerStyle={{
               borderTopWidth: 0.5,
               borderBottomLeftRadius: 8,
               borderBottomRightRadius: 8,
             }}
-            testID="radio/CELO"
+            testID="radio/CREAL"
```

Comment:
```suggestion
            testID="radio/cREAL"
```
nit. matches the other test id's better.